static/coffee/services.coffee

```diff
@@ -698,3 +752,21 @@ services.factory('MessageBoardService', ['$http', '$httpParamSerializer', '$wind
       return $http.post('/messageboardcomment/unpin/' + comment.id + '/')
 
 ])
+
+#=====================================================================
+# InboxUIService service
+#=====================================================================
+services.factory('InboxUIService', ['CaseService', 'ModalService', 'UtilsService', (CaseService, ModalService, UtilsService) ->
```

Comment:
Not convinced this needs to be separate service, rather than in the controller with everything else